### DISCUSSION TITLE
Update GPC_VERSION

### DISCUSSION THREAD

**Discussion by lhf** [rickbrew/GeneralPolygonClipper#5] [Ideas]
If the code is for 2.33, please update GPC_VERSION in gpc.h and the comments in gpc. and gpc.h. Thanks. And for hosting gpc!

**Comment by rickbrew** [comment]
I was maintaining the source as-is, and the 2.33 sources were still stamped with GPC_VERSION of 2.32 and the old 2004 modification timestamp. But, it's confusing that way, and it's harmless to fix. I've updated it!

**Reply by lhf** [reply]
Thanks!
